sdk/rust/oak_tests/src/lib.rs

```diff
@@ -166,6 +166,16 @@ pub fn runtime_config_wasm(
     }
 }
 
+/// Tries to create a `HttpConfiguration`. Returns None, if any of the paths does not represent a
```

Comment:
super-nit: could make `HttpConfiguration` a hyperlink (and maybe move `None` into code font too)